### PR TITLE
Feature/fix allergic reaction translation

### PR DESCRIPTION
**Pull request by matrix-drorgo**
Hi,
I have fixed the translation in allergic stats on both the title and the reaction.
It suppose to look like this (in Hebrew):

image

thnaks
Dror G.

### CODE REVIEW

File: interface/patient_file/summary/stats.php [hunk 1]
```diff
@@ -199,15 +199,19 @@ function load_location( location ) {
...
+
+
             echo " <tr class='text $rowclass;'>\n";
 
             //turn allergies red and bold and show the reaction (if exist)
             if ($key == "allergy") {
                 $reaction = "";
                 if (!empty($row['reaction'])) {
-                    $reaction = " (" . $row['reaction'] . ")";
+                    $reaction= " (" .xlt(ucfirst(str_replace("_"," ",$row['reaction']))) . ")";
```
Comment (bradymiller) [inline interface/patient_file/summary/stats.php]: Isn't the reaction a list from list_options. If so, then should just run through the getListItemTitle() in the library/options.inc.php library.
```
$reaction = " (" . getListItemTitle($row['reaction']) . ")";
```

File: interface/patient_file/summary/stats.php [hunk 2]
```diff
@@ -199,15 +199,19 @@ function load_location( location ) {
...
                 }
 
+                $row['title'] = xlt(ucfirst(str_replace("_"," ",$row['title'])));
```
Comment (bradymiller) [inline interface/patient_file/summary/stats.php]: Aren't users typing this title into a text field. And if so, shouldn't they be entering it via their language?

**Comment by bradymiller** [comment]
hi @matrix-drorgo ,
I left some comments on this above. thanks, -brady

File: interface/patient_file/summary/stats.php (hunk 1 above)
Comment (matrix-drorgo) [reply interface/patient_file/summary/stats.php]: o.k. changed it to : 
           $reaction= " (" .getListItemTitle("reaction",$row['reaction']) . ")";

File: interface/patient_file/summary/stats.php (hunk 2 above)
Comment (matrix-drorgo) [reply interface/patient_file/summary/stats.php]: o.k. removed that row .

**Comment by matrix-drorgo** [comment]
tnx,
fixed the issues above.
Dror G.

**Pull request merged**